uportal-portlets-overlay/CalendarPortlet/pom.xml

```diff
@@ -49,9 +49,9 @@
         </dependency>
 
         <dependency>
-            <groupId>javax.portlet</groupId>
-            <artifactId>portlet-api</artifactId>
-            <version>2.0</version>
+            <groupId>org.apache.portals</groupId>
+            <artifactId>portlet-api_2.1.0_spec</artifactId>
+            <version>1.0</version>
```

Comment:
Interesting.  I wonder if this is still needed for import.  I don't see it passed along to the data-import execution.

Likewise, I can't comment on it for some reason but I wonder if Announcements portlet really needs portlet-api as a dependency for its import/export.